### PR TITLE
🚀  Release v1.8.0-rc.0

### PR DESCRIPTION
**Pull request by vishalanarase**
/area release

### CODE REVIEW

File: CHANGELOG/v1.8.0-rc.0.md [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+- New providers in clusterctl: IONOS Cloud (#10767), Linode (#10471), Tinkerbell (#10487)
+ 
+## Deprecation Warning
+
+- ClusterClass: Deprecate definitionFrom field (#10841)
+- ClusterClass: Deprecate IP Family builtin variable (#10554)
+- MachineDeployment: Deprecate revision management (#10855)
+- MachineHealthCheck: Deprecate MaxUnhealthy & UnhealthyRange fields (#10853)
+
+- Umbrella issue: API changes: #10852
```
Comment (sbueringer) [inline CHANGELOG/v1.8.0-rc.0.md:34]:
```suggestion
Umbrella issue: API changes: #10852
```

File: CHANGELOG/v1.8.0-rc.0.md [hunk 2]
```diff
@@ -0,0 +1,457 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
```
Comment (sbueringer) [inline CHANGELOG/v1.8.0-rc.0.md:3]: Please add:
* #10950
* #10952

Comment (vishalanarase) [reply CHANGELOG/v1.8.0-rc.0.md:3]: Added

Comment (sbueringer) [reply CHANGELOG/v1.8.0-rc.0.md:3]: Sorry the idea wasn't to add these two PRs to highlights (I just put the comment somewhere). Please move them to the correct sections (based on the PR icon)

Comment (sbueringer) [reply CHANGELOG/v1.8.0-rc.0.md:3]: @vishalanarase ^^

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve
/hold

**Comment by adilGhaffarDev** [comment]
LGTM from my side too
@chandankumar4 @Sunnatillo please take a look so I can unhold

**Review by chandankumar4 — APPROVED** [review]
lgtm

**Comment by adilGhaffarDev** [comment]
got multiple lgtms unholding.
/unhold

**Pull request merged**